snuba/datasets/entities/discover.py

```diff
@@ -68,7 +68,7 @@
 from snuba.utils.threaded_function_delegator import Result
 from snuba.web import QueryResult
 
-metrics = MetricsWrapper(environment.metrics, "api.discover.discover_entity")
```

Comment:
Only to match the naming convention of events entity, it's not ideal since we will be resetting the count but I don't think it matters much in this case